packages/core/src/queries/logto-config.ts

```diff
@@ -68,12 +70,29 @@ export const createLogtoConfigQueries = (pool: CommonQueryMethods) => {
       `
     );
 
+  const getJwtCustomizer = async <T extends LogtoJwtTokenKey>(key: T) => {
+    const { rows } = await getRowsByKeys([key]);
+
+    // If the record does not exist (`rows` is empty)
+    if (rows.length === 0) {
+      throw new RequestError({
+        code: 'entity.not_exists',
+        name: table,
+        id: key,
+        status: 404,
+      });
+    }
+
+    return z.object({ value: jwtCustomizerConfigGuard[key] }).parse(rows[0]);
```

Comment:
will this throw a 400 error if the guard fails to parse the value?

Reply:
nit: should we throw an error if rows?length > 1

Reply:
nit: maybe library is a better place for such a method. We should keep the query methods pure.